.github/workflows/ci-osx.yaml

```diff
@@ -1,33 +0,0 @@
-name: OSX CI
-
-on: [push, pull_request]
-
-jobs:
-  build:
-    runs-on: macos-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        python-version: ["3.5", "3.6", "3.7", "3.8"]
-
-    steps:
-      - name: Checkout source
-        uses: actions/checkout@v2
-        with:
-          submodules: recursive
-
-      - name: Set up Python
-        uses: actions/setup-python@v1
-        with:
-          python-version: ${{ matrix.python-version }}
-      
-      - name: Install numcodecs
-        run: |
-          python -m pip install -U pip setuptools wheel pytest
-          python -m pip install -v -e .
-
-      - name: List installed packages
-        run: python -m pip list
-
-      - name: Run tests
-        run: pytest -v --pyargs numcodecs
```

Comment:
My bad. Only remark is that Github actions limit macos concurrent job to 5. 

And wheel are tested using pytest, so it is not only one place.

Reply:
Gotcha. GitHub Action search only works on the logs that you've opened. Happy to have ci-osx.yml re-removed, but I'd propose let's not hold up this since others like @martindurant are having issues due to my 0.7.0 release.